src/doc/The_Configuration_File.md

```diff
@@ -161,7 +161,7 @@ The `client` function configures Ketrew in client-mode:
 
 The `server` function configures the HTTP server:
 
-- The value `` `Tls (certificate, private_key, port)` configures the connection
+- The value `Tls (certificate, private_key, port)` configures the connection
```

Comment:
Using  ``` `` `...  `` ``` is standard in markdown to put backquotes inside code

cf. https://github.com/ocaml/omd/issues/145

but I messed up the closing one, will fix :)